src/modalities.jl

```diff
@@ -0,0 +1,262 @@
+
+abstract type NeuroModality end
+
+
+abstract type AnatomicalImage <: NeuroModality end
+
+struct T1WeightedImage <: AnatomicalImage end
+const T1w = T1WeightedImage()
+modality_label(::T1WeightedImage) = "T1w"
```

Comment:
I originally just had copied a table from the BIDS standard and then did some fancy vim editing to quickly make them into types. I originally intended there to be dispatching on image type for other image processing methods. The `modality_label` argument is a remnant of the prefixes from the BIDS standard that correspond to the modality. I'd actually rather get rid of the `modality_label` and leave the file management/organization to another package completely.

It would probably be better to just have a general modality trait to differentiate anatomical, functional, electrophysiology data. Those could probably be inferred from types at compile time and would be useful for some method dispatch.

Reply:
Right, that sounds simpler.  Agreed that removing `modality_label` may make most sense for now.